test-cases/nemesis/longevity-10gb-3h-AbortRepairMonkey.yaml

```diff
@@ -0,0 +1,24 @@
+test_duration: 255
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=180m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=1000 -pop seq=1..10000000 -log interval=5"
+             ]
+
+n_db_nodes: 6
+n_loaders: 2
+n_monitor_nodes: 1
+
+instance_type_db: 'i4i.2xlarge'
+gce_instance_type_db: 'n1-highmem-16'
+gce_instance_type_loader: 'e2-standard-4'
+azure_instance_type_db: 'Standard_L8s_v3'
+run_fullscan: '{"ks_cf": "keyspace1.standard1", "interval": 5}' # 'ks.cf|random, interval(min)'
+nemesis_class_name: 'AbortRepairMonkey'
+nemesis_seed: '111'
+nemesis_interval: 2
+ssh_transport: 'libssh2'
+
+user_prefix: 'longevity-10gb-3h'
+space_node_threshold: 64424
```

Comment:
Not needed if we have prepare.

but let's add also, nemesis_during_prepare: true.